explainaboard/metrics/external_eval.py

```diff
@@ -37,6 +37,11 @@ class ExternalEvalConfig(MetricConfig):
     n_annotators: int = 3
     categories: int = 5
     instruction: str = "Annotation instruction"
+    # The external statistics for metrics. The row size is equal to the number
+    # of test samples, the column size is equal to `n_annotators`.
+    # NOTE: Use of `list` rather than `numpy.ndarray` is to make this class
+    # serializable.
+    external_stats: list[list[int]] | None = None
```

Comment:
Add `__post_init__` function to validate these conditions.
I'm also worried that maybe double-list is not efficient compared with ndarray in terms of shape checking: ndarray guarantees that every subarray has the same remaining shape, while nested lists does not.